src/eve-server/system/Container.cpp

```diff
@@ -274,6 +276,12 @@ ContainerSE::ContainerSE(CargoContainerRef self, PyServiceMgr& services, SystemM
     m_self->SetAttribute(AttrCapacity, m_self->type().capacity(), false);
 }
 
+void ContainerSE::Init()
+{
+    _log(SE__TRACE, "Cargo %s(%u) is being deployed", m_self->name(), m_self->itemID());
```

Comment:
Might want to swap this for POS__TRACE to keep all of the structure logs in the same place. 

Maybe in the future, we can rename it to STRUCTURE__TRACE or something since not all Structures are POSs, but that's neither here nor there.